src/commcare_cloud/ansible/roles/kafka/tasks/main.yml

```diff
@@ -53,6 +53,23 @@
     dest: "/opt/kafka/bin/kafka-reassign-tool.rb"
     mode: 0755
 
+- name: Ensure new version
+  lineinfile:
+    path: /etc/kafka/config/server.properties
+    regexp: '^inter.broker.protocol.versioni=2.4'
+    state: absent
+    check_mode: yes
+    changed_when: false
+    register: version_2.4_infile
+- name: Ensure old version
+  lineinfile:
+    path: /etc/kafka/config/server.properties
+    regexp: '^inter.broker.protocol.versioni=0.8.2.2'
+    state: absent
+    check_mode: yes
+    changed_when: false
+    register: version_0.8.2.2_infile
+
```

Comment:
i tried checking the values in template , it throws errors the logic i am trying is  
version_2.4_infile.found will retrun true if it matches and version_0.8.2.2_infile.found will retrun true if it's matches